src/osd/scheduler/mClockScheduler.h

```diff
@@ -143,6 +149,18 @@ class mClockScheduler : public OpScheduler, md_config_obs_t {
     };
   }
 
+  static unsigned int get_io_prio_cut(CephContext *cct) {
```

Comment:
cct is already a private member and so you can access it without passing it to the method.

Reply:
You can disregard the above comment. We can let this function be static.